server/v2/cometbft/abci_test.go

```
@@ -669,6 +810,32 @@ func setUpConsensus(t *testing.T, gasLimit uint64, mempool mempool.Mempool[mock.
+	var helloFooHandler = func(ctx context.Context, msg transaction.Msg) (msgResp transaction.Msg, err error) {
+		typedReq := msg.(*testdata.SayHelloRequest)
+		handler := testdata.QueryImpl{}
+		typedResp, err := handler.SayHello(ctx, typedReq)
+		if err != nil {
+			return nil, err
+		}
+
+		return typedResp, nil
+	}
+
+	queryRouterBuilder.RegisterHandler(
```

golangci-lint (1.62.2)

Error return value of `queryRouterBuilder.RegisterHandler` is not checked

(errcheck)